prime-router/metadata/fhir_mapping/hl7/datatype/Identifier_SystemID.yml

```diff
@@ -104,4 +104,25 @@ extension_4:
     value: String, CX.6.1
   constants:
     KEY_NAME_SUFFIX: String
-    urlValue: assigning-facility-namespace-id
\ No newline at end of file
+    urlValue: assigning-facility-namespace-id
+
+# Assigning authority namespace
+extension_5:
+  generateList: true
+  valueOf: extension/Extension
+  expressionType: resource
+  vars:
+    value: String, PID.3.4.1
```

Comment:
Do not specify direct fields in data types.  This must be done at a higher level.  I am pretty sure this needs to be in the value field

Reply:
Not sure what to change here. Let's talk it on monday